Converters/Civil3d/Speckle.Converters.Civil3dShared/ToSpeckle/Properties/ClassPropertiesExtractor.cs

```diff
@@ -76,11 +83,43 @@ PipeNetworkHandler pipeNetworkHandler
     return pointProperties;
   }
 
+  private Dictionary<string, object?> ExtractProfileProperties(CDB.Profile profile)
+  {
+    return new()
+    {
+      ["offset"] = profile.Offset,
+      ["startingStation"] = profile.StartingStation,
+      ["endingStation"] = profile.EndingStation,
+      ["profileType"] = profile.ProfileType.ToString(),
+      ["elevationMin"] = profile.ElevationMin,
+      ["elevationMax"] = profile.ElevationMax
+    };
+  }
+
+  private Dictionary<string, object?> ExtractAlignmentProperties(CDB.Alignment alignment)
+  {
+    Dictionary<string, object?> alignmentProperties =
+      new()
+      {
+        ["startingStation"] = alignment.StartingStation,
+        ["endingStation"] = alignment.EndingStation,
+        ["alignmentType"] = alignment.AlignmentType.ToString()
+      };
+
+    if (!alignment.IsSiteless)
+    {
+      alignmentProperties["siteId"] = alignment.SiteId.GetSpeckleApplicationId();
+    }
+
+    return alignmentProperties;
+  }
+
```

Comment:
We've been intentionally avoiding typed classes because these objects are not intended to be receivable as native objs in any app - just their display values if any, and eventually props